src/Legacy/Content.php

```diff
@@ -216,7 +216,7 @@ public function preParse($snippet, $allowtwig)
         $snippet = html_entity_decode($snippet, ENT_QUOTES, 'UTF-8');
 
         try {
-            return $this->app['safe_render']->render($snippet, $this->getTemplateContext());
```

Comment:
Hmm … this kinda is uncertain. Your sandboxing branch removes it, and I was getting tests across the line for this one, so that yours can land as I know the exact same line changes to (I think) your new `safeRender()`.

We need to check with @bobdenotter as to design intent, as what I discvussed with him yeterday is a valid use-case, but doesn't work the way that is assumed :wink: